### PR TITLE
Pokenav Data Decomp (take 2)

### PR DESCRIPTION
**Pull request by garakmon**

### CODE REVIEW

File: src/data/region_map/city_map_tilemaps.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const u8 gPokenavCityMap_Lavaridge_0[] = INCBIN_U8("graphics/pokenav/city_maps/lavaridge_0.bin.lz");
```
Comment (garakmon) [inline src/data/region_map/city_map_tilemaps.h]: u32

File: src/data/text/trainer_eye_descriptions_en.h [hunk 2]
```diff
@@ -0,0 +1,486 @@
+static const u8 TrainerEyeDescription_0[] = _(
```
Comment (PikalaxALT) [inline src/data/text/trainer_eye_descriptions_en.h]: Name these after the trainers themselves

File: src/pokenav_before.c [hunk 3]
```diff
@@ -104,6 +104,1016 @@ struct UnkPokenavStruct *const gPokenavStructPtr = (struct UnkPokenavStruct *)gS
...
+const struct ScanlineEffectParams gUnknown_083E4990 = 
+{
+    (void *)REG_ADDR_WIN0H,
+    ((DMA_ENABLE | DMA_START_HBLANK | DMA_32BIT | DMA_REPEAT | DMA_DEST_RELOAD) << 16) | 1,
+    1,
+    0,
+};
+
+const u16 gUnknown_083E499C[] = 
+{
```
Comment (PikalaxALT) [inline src/pokenav_before.c]: Use the ribbon constants here

File: src/pokenav_before.c [hunk 4]
```diff
@@ -104,6 +104,1016 @@ struct UnkPokenavStruct *const gPokenavStructPtr = (struct UnkPokenavStruct *)gS
...
+    INCBIN_U16("graphics/misc/ribbons_smart.gbapal"),
+    INCBIN_U16("graphics/misc/ribbons_tough.gbapal"),
+};
+
+const u8 gUnknown_083E3D00[] = INCBIN_U8("graphics/misc/ribbons_full.4bpp.lz");
+const u16 PokenavBlueLightPalette[] = INCBIN_U16("graphics/pokenav/blue_light.gbapal");
+const u8 PokenavBlueLightTiles[] = INCBIN_U8("graphics/pokenav/blue_light.4bpp");
+const u16 Palette_3E42D8[] = INCBIN_U16("graphics/pokenav/83E42D8.gbapal");
+
+const u16 gUnknown_083E42F8[] =
```
Comment (PikalaxALT) [inline src/pokenav_before.c]: Use the `BLDALPHA_BLEND` macro for these

**Comment by garakmon** [comment]
updated

**Pull request merged**